.github/workflows/python.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
```

Comment:
I'd recommend updating the versions of `serverless` and `serverless-offline` in the `python` directory so that `3.10+` can be restored.